pkg/sql/sem/asof/as_of.go

```diff
@@ -132,6 +133,14 @@ func Eval(
 	defer scalarProps.Restore(*scalarProps)
 	scalarProps.Require("AS OF SYSTEM TIME", tree.RejectSpecial|tree.RejectSubqueries)
 
+	// Disable type resolution. Doing that requires a transaction, but this
```

Comment:
nit: "Doing that" is a bit confusing - it sounds like it might be referring to the disabling of type resolution, not that act of type resolution.

Reply:
done